iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -165,7 +165,14 @@ iXBRLViewer.prototype.load = function () {
     setTimeout(function () {
 
         iv._loadInspectorHTML();
-        var iframes = $(iv._reparentDocument());
```

Comment:
Out of curiosity, does the hardcoded `.html` force non-XHTML rendering quirks, or does this `_reparentDocument` already cause those to not work (require the input document to be written to the HTML/XHTML subset)?

Reply:
The line above is from `master`, not the PR branch.   In stub viewer mode, there is no reparent step.  All documents are loaded via the `src` attribute, and I think rendering mode is determined by the content-type set by the web server, which typically follows the file extension.  So I think that even if the stub viewer is `.html`, the main file gets loaded as XHTML if it has a `.xhtml` extension.

In non-stub mode, I think the reparent step above results in the iframe inheriting rendering mode of the parent.